dev/VSIX/ProjectTemplates/Desktop/CSharp/PackagedApp/BlankApp/WinUI.Desktop.Cs.BlankApp.vstemplate

```diff
@@ -2,8 +2,8 @@
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
   <TemplateData>
     <Hidden>true</Hidden>
-    <Name>Blank App (WinUI 3 in Desktop)</Name>
-    <Description>A project for creating a Desktop app based on the Windows UI Library (WinUI 3).</Description>
+    <Name>WinUI Blank App</Name>
```

Comment:
Can we use resx file for Name, description in these templates as well ?

Reply:
good find. we should be using this construct for all item and project templates:
```
    <Name ID="XXX" Package="FIXME-PACKAGEGUID" />
    <Description ID="YYY" Package="FIXME-PACKAGEGUID" />

```
these three need to be localized:
ProjectTemplates/Desktop/CppWinRT/PackagedApp/BlankApp/WinUI.Desktop.CppWinRT.BlankApp.vstemplate
ProjectTemplates/Desktop/CSharp/PackagedApp/WapProj/WinUI.Desktop.Cs.WapProj.vstemplate
ProjectTemplates/Desktop/CppWinRT/PackagedApp/WapProj/WinUI.Desktop.CppWinRT.WapProj.vstemplate

Reply:
ProjectTemplates/Desktop/CSharp/PackagedApp/BlankApp/WinUI.Desktop.Cs.BlankApp.vstemplate too

Reply:
@npuvvada @Scottj1s Fixed!